src/components/Pagination/Pagination.tsx

```diff
@@ -2,0 +1,80 @@
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange?: (page: number) => void;
+};
+
+export const Pagination = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange = () => {},
+}: Props) => {
+  const lastPage = Math.ceil(total / perPage);
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#prev"
+          onClick={() => {
+            if (currentPage > 1) {
+              onPageChange(currentPage - 1);
+            }
+          }}
+          aria-disabled={currentPage === 1 ? 'true' : 'false'}
+        >
+          «
+        </a>
+      </li>
+
+      {getNumbers(1, lastPage).map(number => (
+        <li
+          key={number}
+          className={classNames('page-item', {
+            active: currentPage === number,
+          })}
+        >
+          <a
+            data-cy="pageLink"
+            className="page-link"
+            href={`#${number}`}
+            onClick={() => onPageChange(number)}
+          >
+            {number}
+          </a>
+        </li>
```

Comment:
When rendering a list of components in React, it's important to ensure that the key prop is a stable identifier. If `number` is guaranteed to be unique and stable, this is fine. If not, you might need to find a better key prop.